tests/languages/css/atrule_selector-function-argument_feature.test

```diff
@@ -0,0 +1,24 @@
+@supports selector(::-webkit-foo) {
+	/* style to apply when the `::-webkit-foo` selector is actually supported,
+	 * instead of just being parsed as matching nothing because of Selectors 4
+	 * § Appendix B: https://drafts.csswg.org/selectors-4/#compat */
+}
+
+----------------------------------------------------
+
+[
+	["atrule", [
+		["rule", "@supports"],
+		["function", "selector"],
+		["punctuation", "("],
+		["selector-function-argument", "::-webkit-foo"],

```

Comment:
Why does this produce a different AST?

Reply:
Oh wait, nvm, that's cuz the other is css-extras.

Reply:
Different from the CSS Extras versions? Because CSS Extras adds `inside`, so the selector is tokenized with CSS Extras but not without it.

Reply:
i got it no worries

Reply:
Yeah. I only commented because GitHub doesn't refresh the comments, so I couldn't see your comment while writing mine.